openshift/buildConfig-template.yaml

```diff
@@ -220,3 +220,62 @@ objects:
       triggers:
         - type: ImageChange
           imageChange: {}
+
+  - kind: BuildConfig
+    apiVersion: v1
+    metadata:
+      name: "solver-ubi-8-py36"
+      labels:
+        app: "thoth"
+        component: "solver"
```

Comment:
Do we want to introduce a new label which could be used to query resources of specific solvers (e.g. ubi-8)?

Reply:
We already do it for graph-syncs: https://github.com/thoth-station/graph-sync-job/blob/4fbb4de5d327f432f3be807e71f3290af0dd834d/openshift/inspectionSyncJob-template.yaml#L67

Reply:
ja, good idea